index.html

```diff
@@ -40,6 +40,9 @@
             background: #eeeeee;
             padding: 35px;
         }
+        .content{
+	        overflow:hidden;
```

Comment:
overflow:hidden hide all the content ... 
You have to use this on the pre element. 
```
white-space: pre-wrap; 
word-wrap: break-word;
```

Reply:
gah thanks- sorry!

Reply:
Np